Plugin test cases not ran

1. Break any plugin with tests (for example, remove the `area` key [here](https://github.com/osm-fr/osmose-backend/blob/f8c0579f48d72df366bf8e36fc82f274eba7edb4/plugins/TagFix_Area.py#L67))
2. Create a PR

**Expected:**
Githubs actions report about a failing plugin test (in the example above, I would expect the following)
```
FAILED plugins/TagFix_Area.py::Test::test - AssertionError: {'amenity': 'bakery', 'x': 'no'}
```

**Actual result**
Nothing in `tests` nor `optional_tests` on Github
Example where I tried this (after getting rid of the XML errors): https://github.com/osm-fr/osmose-backend/actions/runs/9921600319/job/27409746231

---


This seems to be related to the configuration of pytest.ini

With configuration `testpaths = . plugins plugins/tests modules analysers`
```
platform linux -- Python 3.11.9, pytest-8.2.2, pluggy-1.5.0
rootdir: /__w/osmose-backend/osmose-backend
configfile: pytest.ini
testpaths: ., plugins, plugins/tests, modules, analysers
plugins: pylama-8.4.1
collected 68 items

plugins/tests/Josm_deprecated_test.py .                                  [  1%]
plugins/tests/Test_mapcss_item_file.py .                                 [  2%]
plugins/tests/test_mapcss_parsing_evaluation.py .                        [  4%]
modules/IssuesFile.py .                                                  [  5%]
```

With configuration `testpaths = . plugins modules analysers`:
```
platform linux -- Python 3.11.9, pytest-8.2.2, pluggy-1.5.0
rootdir: /__w/osmose-backend/osmose-backend
configfile: pytest.ini
testpaths: ., plugins, modules, analysers
plugins: pylama-8.4.1
collected 223 items

plugins/Addr_Interpolation.py .                                          [  0%]
plugins/Administrative_INSEE_Name.py .                                   [  0%]
plugins/Administrative_TooManyWays.py .                                  [  1%]
plugins/Bicycle.py .                                                     [  1%]
plugins/Capacity.py .                                                    [  2%]
plugins/Colour.py .                                                      [  2%]
plugins/ConditionalRestrictions.py F                                     [  3%]
[...]
plugins/indoor.py .                                                      [ 70%]
plugins/notprefix.py .                                                   [ 70%]
modules/IssuesFile.py .                                                  [ 71%]
```

I didn't find a way to enable it for both folder `plugins` and `plugins/tests`. Except when switching back to pytest 7.4.4...

Filed as issue here: https://github.com/pytest-dev/pytest/issues/12605

--- a/requirements.txt
+++ b/requirements.txt
@@ -17,7 +17,7 @@ tiletanic
 sentry-sdk
 
 # Tests
-pytest
+pytest == 7.4.4 # In v8 it skips the plugins folder, see our issue #2266 and https://github.com/pytest-dev/pytest/issues/12605
 xmltodict
 termcolor
 pylama

